gapis/api/vulkan/image_primer.go

```diff
@@ -956,9 +832,9 @@ func (h *ipRenderHandler) render(job *ipRenderJob, tsk *scratchTask) error {
 			[]bufferSubRangeFillInfo{newBufferSubRangeFillInfoFromNewData(sbic.Bytes(), 0)},
 			VkBufferUsageFlagBits_VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT,
 			VkBufferUsageFlagBits_VK_BUFFER_USAGE_TRANSFER_DST_BIT)
-		tsk.deferUntilExecuted(func() {
-			h.sb.write(h.sb.cb.VkDestroyBuffer(dev, stencilIndexBuf, memory.Nullptr))
-		})
+		// tsk.deferUntilExecuted(func() {
```

Comment:
Is this right? (Also if it is, can we remove the commented out code?)

Reply:
@AWoloszyn 
This is right for this CL because scratch buffers will be destroyed automatically after the commands in the task are done.

But I'm also wondering which way is better, 1) destroy automatically, 2) explicitly destroy scratch buffers. Any suggestions?